test/Sentry.Tests/Internals/MemoryInfoTests.WriteTo.Core3_1.verified.txt

```diff
@@ -0,0 +1,8 @@
+{
+  allocated_bytes: 1,
+  fragmented_bytes: 2,
+  heap_size_bytes: 3,
+  high_memory_load_threshold_bytes: 4,
```

Comment:
Could this become flaky?

Reply:
@bruno-garcia i think this test is hard coded values and testing the serialization

Reply:
Yes. Just the values.  The previous test caught my eye because it was doing its own stream processing and then oddly calling Newtonsoft (which we only import transitively), and then it wasn't actually doing anything with the output. So it could have been sending garbage as long as it could serialize and deserialize and we'd never have known it.  Again, it was just a quick fix.  Nothing all that important here.